kano_updater/utils.py

```diff
@@ -585,3 +585,21 @@ def verify_kit_is_plugged():
             ).run()
 
     return is_plugged and not is_battery_low
+
+
+class signalPoll:
```

Comment:
As above, I suggest capitalising the class name `SignalPoll`

Reply:
+1 to above and do you want to split this out into a new file which just contains this class?